cdap-docs/admin-manual/source/_includes/installation/ha-installation.txt

```diff
@@ -0,0 +1,41 @@
+Enabling CDAP HA
+----------------
+In addition to having a :ref:`cluster architecture <admin-manual-install-deployment-architectures-ha>`
+that supports HA (high availability), these additional configuration steps need to followed and completed:
+
+CDAP Master
+...........
+- Install multiple instances of the CDAP Master on different nodes.
+- The instances coordinate amongst themselves, setting a leader and followers.
+
+CDAP Router
+...........
+- Install multiple instances of the CDAP Router on different nodes.
```

Comment:
In the same vein as above,

CDAP Router
CDAP Router is a stateless API endpoint for CDAP, and simply routes requests to the appropriate service.  It can be scaled horizontally for performance.  A load balancer, if desired, can be placed in front of these nodes.
- Install the cdap-router package on different nodes.
- Sync the configuration files (``cdap-site.xml``, ``cdap-security.xml``) on the nodes.  No additional configuration is required for HA.
   - ``router.bind.address`` may need to be customized on each box if not set to the default wildcard address (0.0.0.0).
- Start the cdap-router service on each node.